### ISSUE TITLE
GroupFolder are not shown in "your photo" tab

### ISSUE THREAD

**Issue by bpollet** [nextcloud/photos#575]
Hello, I was not able to see any of my pictures which are stored in a groupfolder. I have to go to sharedfolder to see them but I expected to find them in main photos tab "Your photos"

I saw a prievous ticket here but it is marked as merged:
https://github.com/nextcloud/photos/pull/398
So I'm wondering why I still have this trouble.

I also opened a new ticket on groupfolder app: 
https://github.com/nextcloud/groupfolders/issues/1163#issuecomment-738681761

Nextcloud 20.0.2
Groupfolder: 8.2.0
Photo: 1.2.0
OS: Ubuntu 20.04 64bits
Device: 4GB Raspberry pi4

**Comment by skjnldsv** [comment]
Yes, we don't show groupfolder son the timeline/"your photos" view. We show them on the folder view.
That is related to performance issues as far as I recall.

**Comment by harrellbm** [comment]
Would it be possible to add a settings option to link specific group files to the "your photos" view?  I currently have all my family's photos in one shared folder for easier compiling from multiple phones, etc. but am realizing that it makes add-ons to the photo app (like the location feature) rather useless.  
I would be more than willing to take a crack at it.  I am very new to this project and want to contribute in helpful ways, so any guidance would be appreciated.

**Comment by alexanderdd** [comment]
@skjnldsv sounds like a decision has already been made, so close this as wontfix?

Personally, I would _love_ to have a space where I can see all photos of that cloud sorted by date, my personal ones and the ones shared with me.
A lot of the fotos in groupfolders are actually my photos which I put there, it would make sense to see them in "Your Photos" =)

Maybe this helps with performance: include the ones that are "shared with me on my instance" but not the ones that are "shared with me from other instances (federated)".

**Comment by harrellbm** [comment]
@alexanderdd I've been think about trying to set up an external script through the workflows app to automatically copy over photos from group folders to a user photos folder.  Do you think that would be worth all the extra copies of photos on the server?

**Comment by skjnldsv** [comment]
> @skjnldsv sounds like a decision has already been made, so close this as wontfix?

Yes and no. While we can't right now for performances issues, we should improve the backend to allow us to display the groufolders data in the future.

**Comment by alexanderdd** [comment]
@harrellbm I don't think thats a good idea.

@skjnldsv ok. Does an issue already exist for the necessary backend improvement? If yes please link it here. If no, could you create it?

**Comment by harrellbm** [comment]
@alexanderdd I appreciate the response. Still trying to wrap my head around the nextcloud architecture.

**Comment by cClaude** [comment]
Seem to be link to https://github.com/nextcloud/maps/issues/274 (external storage: photo-scan bug)
